C-Plus-Plus/README.md

```diff
@@ -58,6 +58,7 @@
 - [Moore's Voting Algorithm to find the majority element](cp/moore_voting.cpp)
 - [Ticket Auction](cp/tkt_auction.cpp)
 - [Replace Pi using recursion](cp/replace_pi.cpp)
+- [Lexicographically largest palindrome susequence](cp/Lexicographyically_largest_palindrome_subsequence.cpp)
```

Comment:
Typo Error:`susequence` Kindly fix this.

Reply:
that's `sus`